### ISSUE TITLE
ReferenceError: doneResizing is not defined

### ISSUE THREAD

**Issue by NenadObradovic** [alvarotrigo/multiscroll.js#105]
Hi,

When you destory() / build() slider, you will get this error. Could you please check this? Thanks

Example

```javascript
$(window).resize(function () {
	if ($(window).width() <= 1024) {
		$.fn.multiscroll.destroy();
	} else {
		$.fn.multiscroll.build();
	}
});
```

Nenad

**Comment by alvarotrigo** [comment]
Thanks for reporting it!
I'll mark it as a bug.

If you are in a rush and really need a quick fix, feel free to [contact me](https://alvarotrigo.com/#contact) for custom support.

**Comment by NenadObradovic** [comment]
You are welcome, I will wait new release ;) 

Nenad

**Comment by alvarotrigo** [comment]
Fixed on multiScroll 0.2.2!!
